static/src/js/components/OrderStatus/OrderStatusItem.js

```diff
@@ -455,9 +455,16 @@ export class OrderStatusItem extends PureComponent {
 
             const {
               downloadUrls: currentDownloadUrlsObject = {},
+              // processInfo = {},
               requestStatus = {}
             } = orderInformation
 
+            // processInfo will be where the info messages from EGI are stored
```

Comment:
Leaving this here until that ticket is worked?

Reply:
Yep, added the ticket number on the next line to make it easy to find

Reply:
Excellent, thank you!